### ISSUE TITLE
Python 3.9 Support?

### ISSUE THREAD

**Issue by Mystik01** [brentvollebregt/auto-py-to-exe#140]
Trying to install auto py using PIP:
Get this giant error: https://pastebin.com/QBWgqnDL

The error output:
```
ERROR: Command errored out with exit status 1:
   command: 'c:\program files\python39\python.exe' 'C:\Users\logan\AppData\Roaming\Python\Python39\site-packages\pip' install --ignore-installed --no-user --prefix 'C:\Users\logan\AppData\Local\Temp\pip-build-env-4_7vyg84\overlay' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- 'setuptools >= 40.8.0' wheel 'Cython >= 3.0a5' 'cffi >= 1.12.3 ; platform_python_implementation == '"'"'CPython'"'"'' 'greenlet >= 0.4.17 ; platform_python_implementation == '"'"'CPython'"'"''
       cwd: None
  Complete output (27 lines):
  Collecting setuptools>=40.8.0
    Using cached setuptools-50.3.2-py3-none-any.whl (785 kB)
  Collecting wheel
    Using cached wheel-0.35.1-py2.py3-none-any.whl (33 kB)
  Collecting Cython>=3.0a5
    Using cached Cython-3.0a6-py2.py3-none-any.whl (1.0 MB)
  Collecting cffi>=1.12.3
    Using cached cffi-1.14.3-cp39-cp39-win_amd64.whl (179 kB)
  Collecting greenlet>=0.4.17
    Using cached greenlet-0.4.17.tar.gz (61 kB)
  Collecting pycparser
    Using cached pycparser-2.20-py2.py3-none-any.whl (112 kB)
  Using legacy 'setup.py install' for greenlet, since package 'wheel' is not installed.
  Installing collected packages: setuptools, wheel, Cython, pycparser, cffi, greenlet
      Running setup.py install for greenlet: started
      Running setup.py install for greenlet: finished with status 'error'
      ERROR: Command errored out with exit status 1:
       command: 'c:\program files\python39\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\logan\\AppData\\Local\\Temp\\pip-install-266_sfzw\\greenlet\\setup.py'"'"'; __file__='"'"'C:\\Users\\logan\\AppData\\Local\\Temp\\pip-install-266_sfzw\\greenlet\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\logan\AppData\Local\Temp\pip-record-rjaqo12u\install-record.txt' --single-version-externally-managed --prefix 'C:\Users\logan\AppData\Local\Temp\pip-build-env-4_7vyg84\overlay' --compile --install-headers 'C:\Users\logan\AppData\Local\Temp\pip-build-env-4_7vyg84\overlay\Include\greenlet'
           cwd: C:\Users\logan\AppData\Local\Temp\pip-install-266_sfzw\greenlet\
      Complete output (5 lines):
      running install
      running build
      running build_ext
      building 'greenlet' extension
      error: Microsoft Visual C++ 14.0 is required. Get it with "Build Tools for Visual Studio": https://visualstudio.microsoft.com/downloads/
      ----------------------------------------
  ERROR: Command errored out with exit status 1: 'c:\program files\python39\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\logan\\AppData\\Local\\Temp\\pip-install-266_sfzw\\greenlet\\setup.py'"'"'; __file__='"'"'C:\\Users\\logan\\AppData\\Local\\Temp\\pip-install-266_sfzw\\greenlet\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\logan\AppData\Local\Temp\pip-record-rjaqo12u\install-record.txt' --single-version-externally-managed --prefix 'C:\Users\logan\AppData\Local\Temp\pip-build-env-4_7vyg84\overlay' --compile --install-headers 'C:\Users\logan\AppData\Local\Temp\pip-build-env-4_7vyg84\overlay\Include\greenlet' Check the logs for full command output.
  ----------------------------------------
ERROR: Command errored out with exit status 1: 'c:\program files\python39\python.exe' 'C:\Users\logan\AppData\Roaming\Python\Python39\site-packages\pip' install --ignore-installed --no-user --prefix 'C:\Users\logan\AppData\Local\Temp\pip-build-env-4_7vyg84\overlay' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- 'setuptools >= 40.8.0' wheel 'Cython >= 3.0a5' 'cffi >= 1.12.3 ; platform_python_implementation == '"'"'CPython'"'"'' 'greenlet >= 0.4.17 ; platform_python_implementation == '"'"'CPython'"'"'' Check the logs for full command output.
```

**Comment by brentvollebregt** [comment]
I have added the output to your initial comment - it makes things a lot easier if the text is here.

Did you read the pinned issue #74 for Python 3.8 support?

This doesn't appear to have anything to do with the Python version specifically, it says:

> error: Microsoft Visual C++ 14.0 is required. Get it with "Build Tools for Visual Studio": https://visualstudio.microsoft.com/downloads/

In the help post linked in the README, I have discussed this [issue with installing gevent](https://nitratine.net/blog/post/issues-when-using-auto-py-to-exe/#possible-gevent-issue) - did you give this post a read before making an issue?
